pkg/controller/directvolumemigration/rsync.go

```diff
@@ -553,6 +576,43 @@ func (t *Task) createRsyncTransferPods() error {
 	return nil
 }
 
+func getPodConfig(client k8sclient.Client, cpu_limit string, memory_limit string, cpu_request string, memory_request string, ) (corev1.ResourceList, corev1.ResourceList, error) {
```

Comment:
is there function better named as `getPodResourceLists`?

Reply:
+1